src/middleware/default_headers.rs

```diff
@@ -0,0 +1,70 @@
+use futures::future::FutureObj;
+
+use http::{
+    header::{HeaderValue, IntoHeaderName},
+    HeaderMap, HttpTryFrom,
+};
+
+use crate::{head::Head, Middleware, Response};
+
+#[derive(Clone)]
+pub struct DefaultHeaders {
+    headers: HeaderMap,
+}
+
+// type KeyValues = [&'static str; 2];
+
+impl Default for DefaultHeaders {
```

Comment:
`Default` can be derived for `DefaultHeaders` since `Default` is already impl'd for `HeaderMap` [here](https://docs.rs/http/0.1.13/src/http/header/map.rs.html#1842-1846).

Reply:
Cool! Didn't know you could do that - I'll change that.